docker/README.md

```diff
@@ -1,71 +1,76 @@
-# Deploying the 1Password SCIM bridge using Docker
+# Deploy 1Password SCIM Bridge with Docker
 
-This example describes the methods of deploying the 1Password SCIM bridge using Docker. The Docker Compose and Docker Swarm managers are available and deployment using each manager is described below.
+*Learn how to deploy 1Password SCIM Bridge using Docker Compose or Docker Swarm.*
 
-## Preparing
+**Table of contents:**
 
-Ensure you've read through the [PREPARATION.md](/PREPARATION.md) document before beginning deployment.
+- [Before you begin](#before-you-begin)
+- [Step 1: Choose a deployment option](#step-1-choose-a-deployment-option)
+- [Step 2: Install Docker tools](#step-2-install-docker-tools)
+- [Step 3: Deploy 1Password SCIM Bridge](#step-3-deploy-1password-scim-bridge)
+- [Step 4: Test the SCIM bridge](#step-4-test-the-scim-bridge)
+- [Step 5: Connect your identity provider](#step-5-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Advanced `scim.env` options](#appendix-advanced-scimenv-options)
+- [Appendix: Generate `scim.env` on Windows](#appendix-generate-scimenv-on-windows)
 
-## Docker Compose vs Docker Swarm
+## Before you begin
 
-Using Docker, you have two different deployment options: `docker-compose` and Docker Swarm.
+Before you begin, read the [PREPARATION.md](/PREPARATION.md) document.
```

Comment:
Similarly, it might be nice to standardize this section. AWS says:

"Before you begin, familiarize yourself with [PREPARATION.md](/PREPARATION.md) and complete the necessary steps there."